### ISSUE TITLE
SQL queries can't contain a double quote

### ISSUE THREAD

**Issue by CharlesNepote** [rclement/datasette-dashboards#50]
I'm not sure if it's an issue related to datasette-dashboard or something else in the toolchain but it took me some time to understand my query wasn't working due to double quotes in my `where` clause -- when the YAML file is converted into JSON.

The YAML file was well-formed, but datasette-dashboard wasn't displaying the content of the chart.

Honestly, I don't have any idea how to deal with this in datasette-dashboard. Maybe put a comment in the example:
```yml
        events-source:
          title: Number of events by source
          db: jobs
          # Be careful: don't use double quotes (") inside your query.
          query: SELECT source, count(*) as count FROM events WHERE TRUE [[ AND date >= date(:date_start) ]] [[ AND date <= date(:date_end) ]] GROUP BY source ORDER BY count DESC
```

**Comment by rclement** [comment]
Thanks @CharlesNepote for bringing up this issue. I've never experienced it but I am willing to understand the problem!

A few things to clear things up:
- Could you provide an example of a non-functional query with double-quotes?
- Do you experience the same issue when using Datasette Canned Queries? (https://docs.datasette.io/en/stable/sql_queries.html#canned-queries)

In any case, one need to be careful with quotes in SQL: double and single quotes do not have the same meaning and use.

**Comment by CharlesNepote** [comment]
> Could you provide an example of a non-functional query with double-quotes?

Here is a simple example:
```yml
      charts:
        count:
          title: Total number of measures
          # name of the db without .db
          db: off-stats
          query:
            SELECT count(*) as count FROM data_quality_stats where country == "fr"
          library: metric
          display:
            field: count
            prefix:
            suffix:
```
The datasette instance is working well but I'm entering the /-/dashboards/my-dashboard, the `count` chart is empty and firefox console is yelling: `Uncaught SyntaxError: JSON.parse: expected ',' or '}' after property value in object at line 1 column 182 of the JSON data`
![image](https://user-images.githubusercontent.com/562352/219812892-ae580d58-bf34-4918-8188-0a9a72730c22.png)
If I replace double quote with simple quotes, then it works.

> Do you experience the same issue when using Datasette Canned Queries?

No. The same request with double quotes as a Canned Query is working well.

**Comment by rclement** [comment]
@CharlesNepote Fixed in PR #58
